dali/base/danqs.cpp

```diff
@@ -41,6 +42,7 @@
 // # TYPE nq_active_requests gauge
 // A unsigned scalar should suffice, i.e. never going to have that many active transactions
```

Comment:
the above comment block re. active should be removed as we're not intending to supply this (as can only be 0 or 1) for NQ.

Reply:
Removed.